examples/sysdig-iam-integration/README.md

```diff
@@ -2,70 +2,109 @@
 
-| Name               | Description                                                      | Type         | Default | Required |
-|--------------------|------------------------------------------------------------------|:-------------|:------- |:---------|
-| name               | A descriptive name used to identify the access group             | string       | n/a     | yes      |
-| description        | The description of the access group.                             | string       | n/a     | no       |
-| ag_tags            | Tags that should be applied to the service                       | list(string) | n/a     | no       |
-| roles              | list of roles.                                                   | list(string) | n/a     | yes      |
-| ag_policy_tags     | list of tags that you want to add to the access group policy.    | list(string) | n/a     | no       |
-| account_management | Gives access to all account management services if set to true   | bool         | false   | no       |
-| resources          | A nested block describes the resource of this policy             | string       | n/a     | no       |
-| service_ids        | List of service IDS add to  access group.                        | string       | n/a     | no       |
-| ibm_ids            | IBM IDs that you want to add to or remove from the access group. | list(string) | n/a     | no       |
-
-## Resources Inputs
-
-| Name                          | Description                                                      | Type         | Default | Required |
-|-------------------------------|------------------------------------------------------------------|:-------------|:------- |:---------|
-| service                       | service name that you want to include in your policy definition  | string       | n/a     | no       |
-| resource_instance_id          | ID of resource instance of the policy definition.                | string       | n/a     | no       |
-| region                        | Region of the policy definition                                  | string       | n/a     | no       |
-| resource_type                 | Resource type of the policy definition.                          | string       | n/a     | no       |
-| resource                      | Resource of the policy definition.                               | string       | n/a     | no       |
-| resource_group_id             | ID of the resource group                                         | string       | n/a     | no       |
-| attributes                    | Set resource attributes in the form of name=value,name=value     | string       | n/a     | no       |
+Note: For more information on input varaibles refer https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/iam_access_group_policy
 
 ## Usage
 
-To create access members group run the following command
+To create access group run the following command
```

Comment:
To create access group run the following command

here we have multiple resources change to to create infrastructure (something like that)